server/render/index.js

```diff
@@ -74,8 +75,40 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
+function getCacheKey( context ) {
+	if ( ! isSectionIsomorphic( context.store.getState() ) || context.user ) {
+		return false;
+	}
+
+	if ( isEmpty( context.query ) ) {
+		return context.pathname;
+	}
+
+	let queryParams = Object.keys( context.query );
+	const cacheConfiguration = get( getIsomorphicSectionConfiguration( context.store.getState() ), 'cache', null );
+
+	if ( cacheConfiguration && cacheConfiguration.queryParams === true ) {
+		return context.canonicalPath;
+	}
+
+	if ( cacheConfiguration && Array.isArray( cacheConfiguration.queryParams.exclude ) ) {
+		queryParams = difference( queryParams, cacheConfiguration.queryParams.exclude );
+	}
+
+	if ( isEmpty( queryParams ) ) {
+		return context.pathname;
```

Comment:
If I read this correctly, that if suppose to mean:
> If we have some parameters on this request that are in the exclude list, we shouldn't cache it.

So, shouldn't we return `false` here?
Have I got it wrong?